plugin/hover.py

```diff
@@ -229,8 +259,16 @@ def show_hover(self, listener: AbstractViewListener, point: int, only_diagnostic
     def _show_hover(self, listener: AbstractViewListener, point: int, only_diagnostics: bool) -> None:
         hover_content = self.hover_content()
         contents = self.diagnostics_content() + hover_content + code_actions_content(self._actions_by_config)
-        if contents and not only_diagnostics and hover_content:
-            contents += self.symbol_actions_content(listener, point)
+        link_content, link_has_standard_tooltip = self.document_link_content(listener, point)
```

Comment:
Doesn't seem right to support multiple links but have just a single `link_has_standard_tooltip`. I feel that each link should have its own. So I think it would make more sense if the `document_link_content` would return a list of tuples instead.

That said, having multiple links is probably gonna be such a rare case that I'm not sure how much we should care.

Reply:
Now I've noticed the `len(contents) > 1` logic that sets it to `false` when there are multiple links so I guess it's OK.

Reply:
Yes, the reason is that I append the link to the "symbol action links" now, but only if it is a single link (i.e. from one session) and if it doesn't have a custom title. Just so we know that the title isn't too long, because perhaps there is a server which uses the URL or some other long text as the tooltip/title. Maybe another variable name "can_be_shown_in_symbol_action_links" would be more clearly. But it's a bit long, so...

![Unbenannt](https://user-images.githubusercontent.com/6579999/173431216-f5fa9212-1fe0-4180-bd88-d6491d308dfe.png)